src/components/filter/expert/expert-filter-constants.ts

```diff
@@ -1202,6 +1225,23 @@ export const fields: Record<string, Field[]> = {
         FIELDS_OPTIONS.SUBSTATION_PROPERTY,
         FIELDS_OPTIONS.VOLTAGE_LEVEL_PROPERTY,
     ],
+    DANGLING_LINE: [
```

Comment:
May be after HVDC as in others like criteria, and explicit naming filters?

Reply:
I did a mistake with this comment but it is also right since you have reordered fields' declaration by type..
 The right correction should be here: https://github.com/gridsuite/commons-ui/pull/525#discussion_r1691163970